server/controllers/upgradeController.js

```diff
@@ -0,0 +1,89 @@
+const stripe = require("stripe")(process.env.STRIPE_SK);
+const uuid = require("uuid");
+const Users = require("../models/User");
+const mongoose = require("mongoose");
+
+exports.payment = async function (req, res) {
+  const { product, token } = req.body;
+  console.log("Product", product);
+  console.log("Token", token);
+  //create customer->create subscription->add subscriptionId to User database
+  await stripe.customers
+    .create({
+      email: token.email,
+      source: token.id,
+    })
+    .then((customer) => {
```

Comment:
be careful with these `.then()`s -> you could end up in a callback hell :) 

check this site out: http://callbackhell.com/ 
on ways to prevent this :)

Reply:
I have made the changes and defined separate function. Thanks it would have been hard for me to think of that.